_posts/2017-12-19-container-diff.md

```diff
@@ -0,0 +1,68 @@
+---
+title: RPM support for container-diff
+author: Valentin Rothberg
+layout: blogpost
+---
+
+Google just recently released [container-diff](ihttps://github.com/GoogleCloudPlatform/container-diff) as an open-source project to share it with the containers community. The core use case of container-diff is to analyze container images and report differences among them, including installed software packages with respect to their name, version and size, but it can also analyze files on the images’ file systems.
+
+The release of container-diff instantly caught the attention of the Kubernetes Core Team at SUSE, as we consider it to be a useful tool for the broader containers community, but also for our own open source projects. Analyzing and comparing the differences among containers can be a useful extension to Portus.  While Portus' [security scanning](http://port.us.org/2017/07/19/security-scanning.html) informs users about security issues in images and ultimately protects them from using insecure software, container-diff can assist in managing installed software packages in a more general fashion. For instance, container-diff can be used in automation to find out if or where a given software package is installed, reveal differences among those images, and assist in containerized software management.
```

Comment:
"can be a useful extension to Portus, and we will add support for it in the near future."

Reply:
Updated. Thanks for checking!